flowetl/tests/integration/conftest.py

```diff
@@ -0,0 +1,167 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# -*- coding: utf-8 -*-
+"""
+Conftest for flowetl integration tests
+"""
+import os
+import shutil
+
+from time import sleep
+from subprocess import DEVNULL, Popen
+from pendulum import now, Interval
+from airflow.models import DagRun
+
+import pytest
+import docker
+
+
+@pytest.fixture(scope="session")
+def docker_client():
+    """
+    docker client object
+    """
+    return docker.from_env()
+
+
+@pytest.fixture(scope="session")
+def flowetl_tag():
+    """
+    Get flowetl tag to use
+    """
+    return os.environ.get("FLOWETL_TAG", "latest")
+
+
+@pytest.fixture(scope="module")
+def flowetl_container(docker_client, flowetl_tag):
+    """
+    Fixture that starts a running flowetl container and
+    yields the container object.
+    """
+    container = docker_client.containers.run(
+        f"flowminder/flowetl:{flowetl_tag}", detach=True
```

Comment:
```suggestion
        f"flowminder/flowetl:{flowetl_tag}", detach=True, auto_remove=True
```

Reply:
Just so's I know, why do we want this as a pytest fixture?

Reply:
Doesn't really need to be I suppose...